index.html

```diff
@@ -215,6 +226,34 @@ <h3>Organic Lettuce</h3>
 
             </div>
         </div>
+
+        <hr>
+        <!-- footer section -->
+        <footer>
```

Comment:
Search for footer samples online to get inspiration from. Improve on your footer section to make it more appealing.